src/subdomains/supporting/log/log-job.service.ts

```diff
@@ -167,12 +168,28 @@ export class LogJobService {
   private async getAssetLog(assets: Asset[]): Promise<AssetLog> {
     // custom balance
     const customAssets = assets.filter((a) => Config.financialLog.customAssets?.includes(a.uniqueName));
-    const assetMap = Util.groupBy<Asset, Blockchain>(customAssets, 'blockchain');
+    const customAssetMap = Util.groupBy<Asset, Blockchain>(customAssets, 'blockchain');
 
     const customBalances = await Promise.all(
-      Array.from(assetMap.entries()).map(async ([e, a]) => {
+      Array.from(customAssetMap.entries()).map(async ([e, a]) => {
         const client = this.evmRegistryService.getClient(e);
-        const balances = await this.getCustomBalances(client, a).then((b) => b.flat());
+        const balances = await this.getCustomBalances(client, a, Config.financialLog.customAddresses).then((b) =>
+          b.flat(),
+        );
+        return { blockchain: e, balances };
+      }),
+    );
+
+    // deposit address balance
+    const paymentAssets = assets.filter((a) => a.paymentEnabled);
+    const paymentAssetMap = Util.groupBy<Asset, Blockchain>(paymentAssets, 'blockchain');
+
+    const depositBalances = await Promise.all(
+      Array.from(paymentAssetMap.entries()).map(async ([e, a]) => {
+        const client = this.evmRegistryService.getClient(e);
+        const balances = await this.getCustomBalances(client, a, [
+          EvmUtil.createWallet({ seed: Config.payment.evmSeed, index: 0 }).address,
+        ]).then((b) => b.flat());
```

Comment:
Hm ok, it's a bit more complex here 🤔  we also have Lightning/BTC and Monero/XMR which are payment enabled. Lightning we can ignore, because there is no separate address. But for XMR there is one (see payment config) and we need to get the balance of this address.

Reply:
Hm ok so I would say we have to hardcoded it for monero and Lightning or? 🤔